src/ess/powder/types.py

```diff
@@ -56,9 +56,7 @@ class Filename(sciline.Scope[RunType, str], str):
 pixels by scattering angle into bins given by these edges.
 """
 
-UncertaintyBroadcastMode = Enum(
-    "UncertaintyBroadcastMode", ["drop", "upper_bound", "fail"]
-)
+UncertaintyBroadcastMode = _UncertaintyBroadcastMode
```

Comment:
How is this shown in a graph? In particular in the JSON format. Does it use `ess.powder` or `ess.reduce` as the module?

Reply:
Looks fine?
![image](https://github.com/scipp/essdiffraction/assets/12912489/43c82272-2978-4bf5-b378-f7988d9f9f4e)
... but I don't know if it will still be if we include the module name?

Reply:
> ... but I don't know if it will still be if we include the module name?

That is my point.

But we can leave it as is for now and see how it works out.